.pre-commit-config.yaml

```
@@ -40,20 +49,15 @@ repos:
+            exclude: ^.github/workflows/release.yml$|^.claude
 
     # Temporarily disabled due to issues with cargo-machete while we are stubbing out the project
-    # - repo: https://github.com/bnjbvr/cargo-machete
-    #   rev: v0.9.1
-    #   hooks:
-    #       - id: cargo-machete
+    - repo: https://github.com/bnjbvr/cargo-machete
+      rev: v0.9.2
+      hooks:
+          - id: cargo-machete
+            types: [ cargo, cargo-lock ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)